whois-commons/src/main/java/net/ripe/db/whois/common/InstanceLock.java

```diff
@@ -0,0 +1,68 @@
+package net.ripe.db.whois.common;
+
+import org.apache.commons.lang.StringUtils;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+public class InstanceLock {
+
+    private final static int MAX_INSTANCES = 16;
+    final static String INSTANCE_NAME_PREFIX = "whois-aws-";
```

Comment:
Follow `private static final` order convention

Reply:
The constant is package protected so it can be used by the test.